sys/include/net/netstats.h

```diff
@@ -95,6 +95,9 @@ typedef struct {
 #if IS_USED(MODULE_NETSTATS_NEIGHBOR_LQI) || DOXYGEN
     uint8_t  lqi;           /**< Average LQI of received frames */
 #endif
+#if IS_USED(MODULE_NETSTATS_NEIGHBOR_PHYDAT) || DOXYGEN
```

Comment:
Would it be possible to simply add the parameters instead of driver specific data? Otherwise it breaks the abstraction layer if the upper layer requires a driver specific access

Reply:
The idea was that the upper layer can treat this as opaque data, it would not access it.

Reply:
but what's the reason for having opaque data in an interface? can't it be stored closer to the device?

Reply:
Well I wanted to leverage the hardware feature that RX parameters are the same for TX, so no parsing is needed.
Storing it closer to the device would mean attaching the neighbor statistics to netdev instead of netif, as I understood that was not the way to go.